backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4023,4 +4023,86 @@ databaseChangeLog:
               LEFT JOIN ${database.defaultSchemaName}.result ON t.internal_id=${database.defaultSchemaName}.result.test_event_id 
               WHERE ${database.defaultSchemaName}.result.test_event_id IS NULL AND t.result IS NOT NULL;
       rollback:
-        sql: SELECT 'foo';
\ No newline at end of file
+        sql: SELECT 'foo';
+  - changeSet:
+      id: create-feature-flag-table
+      author: johanna@skylight.digital
+      comment: creates the feature flag table which will host flags that control visibility of features at the application level
+      changes:
+        - tagDatabase:
+            tag: create-feature-flag-table-with-multiplex-flag
+        - createTable:
+            tableName: feature_flag
+            remarks: feature flags to control visibility of features at the application level.
+            columns:
+              - column:
+                  name: internal_id
+                  type: uuid
+                  remarks: The internal database identifier for this entity.
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: text
+                  remarks: The name of the flag. It should describe the feature it represents.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: value
+                  type: boolean
+                  remarks: The value of the flag. It indicates if the feature is enable or disable for the app.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_at
+                  type: DATETIME
+                  remarks: The creation timestamp for this entity.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_by
+                  type: uuid
+                  remarks: The user who created this entity.
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk__anyentity__created_by
+                    references: api_user
+              - column:
+                  name: updated_at
+                  type: DATETIME
+                  remarks: The timestamp for the most recent update of this entity.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: updated_by
+                  type: uuid
+                  remarks: The user who most recently updated this entity.
+                  constraints:
+                    nullable: false
+                    references: api_user
+                    foreignKeyName: fk__anyentity__updated_by
+        - sql: |
+            GRANT SELECT ON ${database.defaultSchemaName}.feature_flag TO ${noPhiUsername};
+        - sql:
+            remarks: Insert multiplexEnabled flag with value equal true.
```

Comment:
Can you create a separate changeSet for this? I'm not totally sure how liquibase will execute multiple sql statements in the same changeset. It will also make potential rollbacks easier, so that we can remove the multiplex flag without needed to drop the entire table. (Unlikely use case, but it's a good practice nevertheless)